src/main/java/handlers/MethodOptions2Handler.java

```diff
@@ -0,0 +1,50 @@
+package main.java.handlers;
+
+import main.java.server.HTTPMessageComponents.HTTPMethods;
+import main.java.server.response.Response;
+import main.java.server.request.Request;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+import static main.java.server.HTTPMessageComponents.HTTPSyntax.*;
+import static main.java.server.HTTPMessageComponents.StatusCodes.NOT_ALLOWED;
+import static main.java.server.HTTPMessageComponents.StatusCodes.OK;
+
+public class MethodOptions2Handler implements Handler {
+
+    @Override
+    public Response generateResponseForRequest(Request request) {
+        if (!methodIsAllowed(request.getRequestMethod())) {
+            return new Response(VERSION + SP + NOT_ALLOWED + CRLF, buildHeaders());
+        }
+        return new Response(VERSION + SP + OK + CRLF, buildHeaders());
+    }
+
+    @Override
+    public String getPath() {
+        return "/method_options2";
+    }
+
+    @Override
+    public String[] allowedMethods() {
+        return new String[]{HTTPMethods.GET,
+                            HTTPMethods.HEAD,
+                            HTTPMethods.OPTIONS,
+                            HTTPMethods.PUT,
```

Comment:
Probably not something we need to tackle with this story, but I'm wondering what would happen if I actually hit this route with `PUT` or `POST`